jupyter-cil-notebook/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# sources:
+# - https://github.com/jupyter/docker-stacks
+# - https://github.com/TomographicImaging/CIL#installation-of-cil
+# TODO: use `ubuntu-22.04` once `cil` adds `python=3.11` support
```

Comment:
AFAIK, we follow https://numpy.org/neps/nep-0029-deprecation_policy.html to drop support, but I am not sure about adding support version of python.

Reply:
superseded by https://scientific-python.org/specs/spec-0000/ (which incidentally means py3.11 should've been fully supported since last year)

Reply:
Can you open an issue on [CIL](https://github.com/TomographicImaging/CIL), please

Reply:
https://github.com/TomographicImaging/CIL/issues/1490